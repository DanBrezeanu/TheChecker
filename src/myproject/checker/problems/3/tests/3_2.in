7
6572 6229 2561 6290 7924 496 7715 
