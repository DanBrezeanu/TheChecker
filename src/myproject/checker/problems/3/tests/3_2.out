496 6290 6572 7924 
7715 6229 2561 
